python/pyspark/sql/dataframe.py

```diff
@@ -363,9 +363,18 @@ def schema(self) -> StructType:
 
         Examples
```

Comment:
Can we add `Returns` section?

Reply:
Let's add `Returns` section since we're here.

Reply:
I think the returns section is already there..

![image](https://user-images.githubusercontent.com/89562186/186796266-2fef2771-d79c-45c6-a99c-4883a8714000.png)